### ISSUE TITLE
Still getting Jade heat frame bug

### ISSUE THREAD

**Issue by Wgairborne** [TeamPneumatic/pnc-repressurized#1156]
### Describe the bug

Similar to issue #1125 , putting a heat frame on an inventory shows this error:

```
04/11/2023 - 23:37:40
me.desht.pneumaticcraft.common.thirdparty.waila.SemiblockProvider$ComponentProvider@2d9e2eaa
java.lang.AbstractMethodError: Receiver class me.desht.pneumaticcraft.common.entity.semiblock.HeatFrameEntity does not define or inherit an implementation of the resolved method 'abstract net.minecraft.network.chat.Component getDisplayName()' of interface me.desht.pneumaticcraft.api.semiblock.ISemiBlock.
	at TRANSFORMER/pneumaticcraft@1.19.2-4.3.3-22/me.desht.pneumaticcraft.common.thirdparty.waila.SemiblockProvider$ComponentProvider.appendTooltip(SemiblockProvider.java:77)
	at TRANSFORMER/jade@8.8.1/snownee.jade.impl.BlockAccessorImpl._gatherComponents(BlockAccessorImpl.java:120)
	at TRANSFORMER/jade@8.8.1/snownee.jade.overlay.WailaTickHandler.tickClient(WailaTickHandler.java:127)
	at TRANSFORMER/jade@8.8.1/snownee.jade.util.ClientPlatformProxy.onClientTick(ClientPlatformProxy.java:138)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:260)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:252)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:315)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:296)
	at TRANSFORMER/forge@43.2.0/net.minecraftforge.event.ForgeEventFactory.onPostClientTick(ForgeEventFactory.java:830)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.Minecraft.m_91398_(Minecraft.java:1850)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1078)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.Minecraft.m_91374_(Minecraft.java:700)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.main.Main.m_239872_(Main.java:212)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.client.main.Main.main(Main.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at MC-BOOTSTRAP/fmlloader@1.19.2-43.2.0/net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
```

Heat frame still functions correctly from what I see.

### How to reproduce the bug

Put a heat frame on an inventory.

### Expected behavior

For Jade not to mention an error occurring, and to not write an error output.

### Additional details

Running Forge 43.2.0 on 1.19.2

### Which Minecraft version are you using?

1.19

### Which version of PneumaticCraft: Repressurized are you using?

1.19.2-4.3.3-22

**Comment by Selbram** [comment]
I am getting this same error in the UniverseIO modpak except the Heat frame is not working. It will attach to an inventory but shows an error when you hover over it with waila. The text is the same in the logs as posted by OP.

**Comment by desht** [comment]
Should be properly fixed now in 4.3.4
